server/engine/src/main/java/org/eclipse/lsp/cobol/implicitDialects/cics/utility/CICSOptionsCheckBaseUtility.java

```diff
@@ -375,6 +375,34 @@ protected int checkHasMutuallyExclusiveOptions(String options, List<TerminalNode
         }
         return nodes.size();
     }
+    /**
+     * Flags errors for rule lists passed as parameters if there are multiple instances of mutually
+     * exclusive options.
+     *
+     * @param rules   Lists of TerminalNode to iterate through
+     * @return Number of TerminalNode instances found
+     */
+    protected int checkHasMutuallyExclusiveOptions(List<TerminalNode>... rules) {
```

Comment:
Why not just use the one above with a static string like it is done in all other cases?